src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+ abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();

```

Comment:
add an empty line between these methods. run `mvn clean package` before creating the PR, this code is not pass the checkstyle step. you can also see this on github  in tab `Checks`